internal/tool/multi/deployer.go

```diff
@@ -353,8 +353,10 @@ func (d *deployer) startColocationGroup(g *group) error {
 			d.stop(err)
 			return err
 		})
-		if err := d.registerReplica(g, wlet, e.Pid()); err != nil {
-			return err
+		if pid, ok := e.Pid(); ok {
```

Comment:
What do you think about panicking if `!ok`? If I understand correctly, the multi deployer should never create an `Envelope` with an in-process `Child`, so `ok` should always be true. And if `ok` isn't true and we don't call `registerReplica`, I think the deployer may end up in an unexpected state and either crash or hang. 

And same for the SSH deployer.

Reply:
Done